File: iniciante/python/1113-crescente-e-decrescente.py
def crescente_e_decrescente():
    while True:
        n1, n2 = entrada()
        if n1 > n2:
            print('Decrescente')
        elif n1 < n2:
            print('Crescente')
        else:
            break


def entrada():
    numeros = list(map(int, input().split(' ')))
    numero1 = numeros[0]
    numero2 = numeros[1]
    return numero1, numero2


crescente_e_decrescente()

File: iniciante/c/1113-crescente-e-decrescente.c
#include <stdio.h>
#include <stdbool.h>

int main(){
    int x, y;
    while (true){
        scanf("%d %d", &x, &y);
        if (x > y){
            printf("Decrescente\n");
        }
        else if (x < y){
            printf("Crescente\n");
        }
        else{
            break;
        }
    }
    return 0;
}
